app/packages/looker/src/worker/index.ts

```
@@ -259,18 +188,27 @@ const processLabels = async (
     const cls = getCls(`${prefix ? prefix : ""}${field}`, schema);
 
+    if (!cls) {
+      continue;
+    }
+
     for (const label of labels) {
       if (!label) {
         continue;
       }
       if (painterFactory[cls]) {
+        if (sample.id.endsWith("50a4")) debugger;
```

Biome (1.9.4)

[error] This is an unexpected use of the debugger statement.

Unsafe fix: Remove debugger statement


(lint/suspicious/noDebugger)